tests/unittests/BackendTest.cpp

```diff
@@ -497,3 +497,13 @@ INSTANTIATE_TEST_CASE_P(JIT, BackendTest, ::testing::Values(BackendKind::CPU));
 INSTANTIATE_TEST_CASE_P(OpenCL, BackendTest,
                         ::testing::Values(BackendKind::OpenCL));
 #endif // GLOW_WITH_OPENCL
+
+// check if the dump function works.
```

Comment:
doxygen comments everywhere.

Reply:
> doxygen comments everywhere.

OK. I added the doxygen comments to all the .h file and test functions to explain the defination and test goal.